### PR TITLE
Fix tough-cookie usage example

### PR DESCRIPTION
**Pull request by luismesejo**
#### Checklist

Fixes https://github.com/sindresorhus/got/issues/835

Just some documentation update. The sample code for the tough-cookie library usage was throwing some errors...

- [x ] I have read the documentation.
- [ x] I have included a pull request description of my changes.
- [ ] I have included some tests.
- [ ] If it's a new feature, I have included documentation updates.

### CODE REVIEW

**Comment by sindresorhus** [comment]
It should use `util.promisify`.

**Comment by luismesejo** [comment]
so, I tried as @szmarczak suggested in https://github.com/sindresorhus/got/issues/835, and it's crashing... probably I'm doing something wrong..

**Comment by sindresorhus** [comment]
Crashing with what error? Make sure you copy-pasted exactly what @szmarczak commented.

**Comment by luismesejo** [comment]
works, udpated pr

File: readme.md [hunk 1]
```diff
@@ -950,12 +950,15 @@ You can use the [`tough-cookie`](https://github.com/salesforce/tough-cookie) pac
 
 ```js
 const got = require('got');
-const {CookieJar} = require('tough-cookie');
-
-const cookieJar = new CookieJar();
-cookieJar.setCookie('foo=bar', 'https://www.google.com');
-
-got('https://google.com', {cookieJar});
+const { CookieJar } = require('tough-cookie');
```
Comment (szmarczak) [inline readme.md]: Follow the style: `{CookieJar}`

File: readme.md [hunk 2]
```diff
@@ -950,12 +950,15 @@ You can use the [`tough-cookie`](https://github.com/salesforce/tough-cookie) pac
...
+const util = require('util');
+
+const yourFunction = async () => {
```
Comment (szmarczak) [inline readme.md]: You don't need to name the function. Just execute it immediately:

```js
(async () => {
    ...
})();
```

Comment (luismesejo) [reply readme.md]: that throws SyntaxError: await is only valid in async function

Comment (luismesejo) [reply readme.md]: ok

File: readme.md (hunk 1 above)
Comment (luismesejo) [reply readme.md]: ok

**Pull request merged**